src/components/ImageView/ImageViewComponent.tsx

```diff
@@ -225,7 +225,10 @@ export class ImageViewComponent extends React.Component<WidgetProps> {
 
         return (
             <ReactResizeDetector handleWidth handleHeight onResize={this.onResize} refreshMode={"throttle"} refreshRate={33}>
-                <div className="image-view-div" style={{gridTemplateColumns: `repeat(${appStore.numImageColumns}, auto)`}}>
```

Comment:
@crocka are we able to tell when it is "auto", what's the width and height in px?